docs/reference/release-notes/8.14.1.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[release-notes-8.14.1]]
+== {es} version 8.14.1
+
+coming[8.14.1]
```

Comment:
```suggestion

```

Reply:
Is this a bug in our release notes generation

Reply:
I think it would be fine except this PR is coming after the release is already live.

Reply:
So if the timeline is that these PRs will continue to be created on release day, then yes it ought to be removed.

Reply:
Our release automation would have taken care of it after this was merged. It's okay to remove it in this PR, but it wasn't necessary